docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -262,6 +267,53 @@ docker run -i --rm -p 8080:8080 quarkus-quickstart/getting-started
 
 NOTE: Interested by tiny Docker images, check the {quarkus-images-url}/graalvm-{graalvm-version}/distroless[distroless] version.
 
+== Producing a container with a multi-stage Docker build
+
+On the previous paragraph, we show you how to build a native image with Maven, however this implies that GraalVM is installed locally, and if you use
+a CI/CD pipeline, inside your CI/CD.
```

Comment:
```suggestion
a CI/CD pipeline, installed inside your CI/CD.
```
It think it's a bit clearer. I had a parse error when I first read this sentence.

Reply:
Although the meaning is conveyed, this doesn't sound like it idiomatic English. 
I would rather phrase it like:

The previous sectioned showed you how to build a native image using Maven, but implicitly required that the proper GraalVM version be installed on the building machine (be it your local machine on your CI/CD infrastructure).

Reply:
done

Reply:
My mistake here... I wrote `sectioned` instead of `section`, can you please fix it?

Reply:
I saw it, already corrected ;)

Reply:
Thanks!